lib/util/index.js

```diff
@@ -520,6 +520,40 @@ function valueToJSON(value) {
   return value;
 }
 
+/**
+ * Create the bandwidth profile payload included in an RSP connect message.
+ * @param {BandwidthProfileOptions} bandwidthProfile
+ * @returns {object}
+ */
+function createBandwidthProfilePayload(bandwidthProfile) {
+  const payload = {};
+  if (typeof bandwidthProfile.video !== 'object') {
+    return payload;
+  }
+
+  const mode = bandwidthProfile.video.mode;
+  if (typeof mode === 'string') {
+    payload.mode = mode;
+  }
+
+  const maxSubscriptionBitrate = bandwidthProfile.video.maxSubscriptionBitrate;
+  if (typeof maxSubscriptionBitrate === 'number') {
+    payload.video = Object.assign({
```

Comment:
do we need `Object.assign` here? Seems like 
```
payload.video = {
   max_subscription_bandwidth: Math.round(maxSubscriptionBitrate / 1024)
};
```
would just do the job, and is more readable.

Reply:
@makarandp0 This would overwrite other properties in `payload.video`.

Reply:
but given that  payload.video is `undefined` before this assignment we do not have to worry about its existing values. IMO Something like this would look simpler. But now I kind of understand your motivation to use `Object.assign`
```
payload.video = {};
if (typeof bandwidthProfile.video.maxSubscriptionBitrate === 'number') {
  payload.video.max_subscription_bandwidth =  Math.round(bandwidthProfile.video.maxSubscriptionBitrate / 1024)
}

if (typeof bandwidthProfile.video.maxTracks === 'number') {
  payload.video.maxTracks = bandwidthProfile.video.maxTracks;
}
```

Reply:
@makarandp0 In your example, The `.video` property is created even when both `maxSubscriptionBitrate` and `maxTracks` are not present, which shouldn't happen. So, using `Object.assign()` is the most straight-forward way.